src/controller_examples/rabbitmq_controller/proof/liveness/helper_invariants/invariants.rs

```diff
@@ -784,8 +1047,8 @@ pub proof fn lemma_true_leads_to_always_every_create_cm_req_does_the_same(spec:
                         reveal_strlit("-server-conf");
                         reveal_strlit("-plugins-conf");
                         if str1.len() == str2.len() {
-                            assert(str1[str1.len() - 6] == 's');
-                            assert(str2[str1.len() - 6] == 'r');
+                            assert(str1[str1.len() - 6] == 'r');
+                            assert(str2[str1.len() - 6] == 's');
```

Comment:
Did these assertions pass before?

Reply:
You know the CI always passes before we merge... So... Interesting.

Reply:
omg, sounds like some soundness issue of Verus?
How did you find this problem?

Reply:
If there is no soundness issue, the only possible reason is that Verus thinks `str1.len() == str2.len()` can never be true, somehow. You can verify this by adding a `assert(false);` inside this branch

Reply:
> omg, sounds like some soundness issue of Verus?

I don't think so. Maybe this is because it successfully proves the post condition before testing the assertions? Then it just skips.

Reply:
> How did you find this problem?

This pattern is used in several places. It fails in another place. Since I basically did copy and paste, so I found it.

Reply:
> > How did you find this problem?
> 
> This pattern is used in several places. It fails in another place. Since I basically did copy and paste, so I found it.

LOL

Reply:
> Then you can try to remove the assertions?

I removed it then the `assert_by` failed... 😨

Reply:
We should save this branch and report it to the Verus team.